src/viam/app/data/client.py

```diff
@@ -44,21 +47,16 @@ def __init__(self, channel: Channel, metadata: Mapping[str, str]):
     _data_sync_client: DataSyncServiceStub
     _metadata: Mapping[str, str]
 
-    async def tabular_data_by_filter(
-        self,
-        filter: Optional[Filter],
-        dest: Optional[str]
-    ) -> List[Mapping[str, Any]]:
+    async def close(self):
```

Comment:
I think we don't need this, since `close` is now being handled by the `AppClient`.